Azure/cloud-init.txt

```diff
@@ -0,0 +1,166 @@
+apt-key fingerprint 0EBFCD88
+
+add-apt-repository \
+	"deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+	$(lsb_release -cs) \
+	stable"
+
+apt-get update
+apt-get install -y docker-ce
+
+# Install tests dependencies
+apt-get install -y \
+	bc \
+	smem \
+	apache2-utils \
+	iperf
+
+# Install thin tools for devicemapper configuration
+apt-get install -y thin-provisioning-tools
+
+# Configure the clear containers runtime using devicemapper as
+# storage driver
+mkdir -p /etc/systemd/system/docker.service.d/
+cat << EOF | sudo tee /etc/systemd/system/docker.service.d/clr-containers.conf
+[Service]
+ExecStart=
+ExecStart=/usr/bin/dockerd -D --add-runtime cc-runtime=/usr/bin/cc-runtime --default-runtime=cc-runtime
+EOF
+
+# Add devicemapper configuration
+mkdir -p /etc/docker
+
+mkdir -p /etc/lvm/profile
+cat << EOF | sudo tee /etc/docker/daemon.json
+{
+"storage-driver": "devicemapper",
+"storage-opts": [
+        "dm.directlvm_device=/dev/sdc",
+        "dm.thinp_percent=95",
+        "dm.thinp_metapercent=1",
+        "dm.thinp_autoextend_threshold=80",
+        "dm.thinp_autoextend_percent=20",
+        "dm.directlvm_device_force=true"
+]
+}
+EOF
+
+# Load devicemapper changes
+systemctl daemon-reload
+
+# Update kernel
+echo deb http://archive.ubuntu.com/ubuntu/ xenial-proposed restricted main multiverse universe >> /etc/apt/sources.list
+apt-get update
+apt-get install -y linux-azure
+
+# Install Kubernetes and kubeadm
+apt-get install -y ebtables ethtool
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+cat << EOF > /etc/apt/sources.list.d/kubernetes.list
+deb http://apt.kubernetes.io/ kubernetes-xenial main
+EOF
+apt-get update
+sudo -E apt install -y kubelet=1.7.8-00 kubeadm=1.7.8-00 kubectl=1.7.8-00
+
+# Create kubelet systemd service
+cat << EOF | sudo tee /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+[Service]
+Environment="KUBELET_KUBECONFIG_ARGS=--kubeconfig=/etc/kubernetes/kubelet.conf --require-kubeconfig=true"
+Environment="KUBELET_SYSTEM_PODS_ARGS=--pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true"
+Environment="KUBELET_NETWORK_ARGS=--network-plugin=cni --cni-conf-dir=/etc/cni/net.d --cni-bin-dir=/opt/cni/bin"
+Environment="KUBELET_DNS_ARGS=--cluster-dns=10.96.0.10 --cluster-domain=cluster.local"
+Environment="KUBELET_AUTHZ_ARGS=--authorization-mode=Webhook --client-ca-file=/etc/kubernetes/pki/ca.crt"
+Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --container-runtime-endpoint=/var/run/crio.sock --runtime-request-timeout=30m"
+Environment="KUBELET_CADVISOR_ARGS=--cadvisor-port=0"
+ExecStart=
+ExecStart=/usr/bin/kubelet \$KUBELET_KUBECONFIG_ARGS \$KUBELET_SYSTEM_PODS_ARGS \$KUBELET_NETWORK_ARGS \$KUBELET_DNS_ARGS \$KUBELET_AUTHZ_ARGS \$KUBELET_CADVISOR_ARGS \$KUBELET_EXTRA_ARGS
+Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --container-runtime-endpoint=/var/run/crio.sock --runtime-request-timeout=30m"
```

Comment:
@MarioCarrilloA One more fix: this line here is a leftover, it's not needed. Could you please remove it ?